modules/pico_dev_ppp.c

```diff
@@ -302,6 +302,10 @@ static void lcp_optflags_print(struct pico_device_ppp *ppp, uint8_t *opts, uint3
 #define PPP_TIMER_ON_AUTH       0x10u
 #define PPP_TIMER_ON_IPCP       0x20u
 
+static int should_escape(uint8_t byte) {
+  return (byte == PPPF_FLAG_SEQ || byte == PPPF_CTRL_ESC || byte < 0x20);
+}
+
```

Comment:
@thijsdv probably knows more about PPP than I do.
Thijs, could you have a look whether this modification makes sense to you as well?